### DISCUSSION TITLE
Can I use @rainbow-me/rainbowkit on non-standard web pages, such as in a chrome plug-in I developed?

### DISCUSSION THREAD

**Discussion by maicFir** [rainbow-me/rainbowkit#1674] [Q&A] [answered]
I want to use it on non-standard web pages, such as using @rainbow-me/rainbowkit in the settings page of the chrome plug-in I developed. I found that it cannot be detected that my browser has installed the metaMask plug-in. How to solve it?
![Snipaste_2023-12-29_17-01-27](https://github.com/rainbow-me/rainbowkit/assets/21973334/7ff4feea-dbb7-44b4-a9e6-e2fccdb8020f)

**Comment by DanielSinclair** [comment, marked as answer]
@maicFir That behavior is a limitation of Chrome extension sandboxing, unfortunately. The better path forward would be to deploy a separate website and link to that from your local Chrome Extension bundle.

**Reply by maicFir** [reply]
Thank you very much for your reply
